backend/app/service.py

```diff
@@ -68,6 +118,14 @@ def login(data: OAuth2PasswordRequestForm = Depends(), db: Session = Depends(get
     return {"access_token": access_token}
 
 
+@app.get("/users", response_model=List[schemas.User], tags=["users"])
+def get_all_users(user: User = Depends(manager), db: Session = Depends(get_db)):
+    if user.is_moderator_or_more():
+        print(ops_user.get_users(db))
```

Comment:
Forgotten print

Reply:
Please structure the calls in service in the way that we check the permissions and raises exception as if statement. The main "body" should be "happy path". i.e.:

```python
if not have_permissions:
    raise ...
doSomethingProtected()
```